include/rapidjson/reader.h

```diff
@@ -1041,16 +1044,21 @@ class GenericReader {
             else if (RAPIDJSON_UNLIKELY(static_cast<unsigned>(c) < 0x20)) { // RFC 4627: unescaped = %x20-21 / %x23-5B / %x5D-10FFFF
                 if (c == '\0')
 					RAPIDJSON_PARSE_ERROR(kParseErrorStringMissQuotationMark, is.Tell());
-				else if (!isKey && c == '\n' && (parseFlags & kParseMultiLineValueFlag)) {
+				else if (!isKey && c == '\n' && (parseFlags & kParseMultiLineStringValueFlag)) {
```

Comment:
same indent issues

Reply:
This looks normal locally, I'm not sure why it's showing like this on github??